free-2.md

```diff
@@ -5,51 +5,75 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project would serve game developers and designers, especially ones with fewer experience in coding and newer to this idea, as it would make game development faster, easier, and more efficient.
+
 ### What's the need?
 
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+With this idea, the need of specifying game logic, levels, characters, and any other elements of game designing would be made much easier for developers. I would help people with fewer experience in general-purpose programming languages, as they do not know many things and the complicated things of a GPPL to create the game of their dreams. With this language, their experience would be much easier to create a game as it would reduce the time and effort required to build a game.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because it helps game developers create a game without knowing too much of a different language, as it addresses this need by having its own, easier to understand language than a general-purpose programming language. After developers get familiar with this language, they would be able to create games much easier, hence a DSL is appropriate for this area.
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I am very into video games, and also game programming. In fact, I am part of an Esports team in Fifa 23, a video game for soccer. My first experience with coding has actually been with game programming, so I have always dreamt of building my own game. When I first took this class, I instantly thought of this domain, so that is how I came up with this idea.
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Easier game development and design
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+The user would have to get familiar with this language. It may be similar to a host language, so users with experience with the host language may have an advantage. The programming would, however, have its own syntax that will help ease creating the game logic, levels, etc. I believe this is the right way to interact with this domain because fully knowing a general-purpose programming language to design a game is almost impossible, and may end up with developers not being able to fully execute their dreams. I believe if they fully understand this language, they can do whatever they want in their games.
```

Comment:
What might be an example of this dsl with regards to game design like creating levels?